src/io/iohk/scalanet/peergroup/SimplePeerGroup.scala

```diff
@@ -67,11 +75,12 @@ class SimplePeerGroup[A, AA](
 
       val enrolledTask: Task[Unit] = underLyingPeerGroup
         .messageChannel[Enrolled[A, AA]]
-        .map { enrolled =>
-          routingTable.clear()
-          routingTable ++= enrolled.routingTable
-          log.debug(s"Peer address '$processAddress' enrolled into group and installed new routing table:")
-          log.debug(s"${enrolled.routingTable}")
+        .map {
+          case (_, enrolled) =>
+            routingTable.clear()
+            routingTable ++= enrolled.routingTable
+            log.debug(s"Peer address '$processAddress' enrolled into group and installed new routing table:")
```

Comment:
Use a single log to make it easy to locate them on concurrent environments.

Reply:
+1